resources/datasets/hmda/generate_msamd_csv.sh

```diff
@@ -151,12 +151,30 @@ sed_tweaks='
 
 if [ $INPUT_TYPE = 'xlsx' ]; then
     # xlsx2csv gracefully handles unicode nastiness, thankfully.
-    dump_cmd="xlsx2csv -d '|' \"$1\""
+    dump_cmd="xlsx2csv -d '|' \"$INPUT_FILE\""
+elif [ $INPUT_TYPE = 'tab' ]; then
+    # this was specifically made for FFIEC files https://www.ffiec.gov/hmda/hmdaflat.htm 
+    # (https://www.ffiec.gov/hmdarawdata/OTHER/2007HMDAMSADescription.zip)
+    # Convert DOS end of line characters with characters awk will properly handle
+    dos2unix $INPUT_FILE
+    if [ $? -ne 0 ]; then
+        echo 'install dos2unix with `brew install dos2unix` in order to use tab-delimited files'
+        echo 'exiting'
+        exit 1
+    fi
+    # Must use gsed to properly detect tab characters (on OSX at least)
+    if [ $sed_bin != 'gsed' ]; then
```

Comment:
gsed isn't a "thing" in linux land, regular sed is the same thing, so a blanket requirement to require gsed doesn't make sense.

Also, this check is required for the entire script, not just the 'tab' section.  How about we improve the check in 103 to check whether or not gsed is actually required?  One way I *think* we can do this is if `sed --version